index.html

```diff
@@ -1453,45 +1444,13 @@
 
 </pre>
         <p>
-          The <dfn for="Presentation"><code>defaultRequest</code></dfn> MUST
-          return the <a>default presentation request</a> if any, null
-          otherwise.
+          The <a>PresentationReceiver</a> object is available to a <a>presenting
```

Comment:
In the spirit of keeping commits focused on one thing, I will do that in a follow-up.